rat/model.py

```diff
@@ -33,7 +33,7 @@ class Model:
     parameter_offsets: List[int]
     parameter_sizes: List[Union[None, int]]
 
-    def constrain(self, unconstrained_parameter_vector, pad=True):
+    def _constrain(self, unconstrained_parameter_vector, pad=True):
```

Comment:
I added an underscore to a bunch of these internal functions so they won't show up as external.